src/applications/vaos/tests/mocks/data.js

```diff
@@ -0,0 +1,227 @@
+/**
+ * @module testing/mocks/data
+ */
+import moment from '../../lib/moment-tz';
+import omit from 'platform/utilities/data/omit';
+import { VIDEO_TYPES } from '../../utils/constants';
+
+/**
+ * Creates a mock appointment record, based on the data and version number
+ * provided.
+ *
+ * Versions currently supported:
+ * - 0: MAS appointment
+ * - 2: VAOS service appointment
+ *
+ * @export
+ * @param {Object} params
+ * @param {?string} params.id The appointment id
+ * @param {?string} params.email The email address used
+ * @param {?string} params.currentStatus The VistA status to use. Ignored in version 2
+ * @param {Number} [params.version=2] The version of the output data. Currently 0 and 2 are supported
+ * @param {?string} params.clinicFriendlyName The clinic name of the appointment (version 0 only)
+ * @param {?string} params.clinicName The regular clinic name of the appointment (version 0 only)
+ * @param {?string} params.instructionsTitle The video instructions title string
+ * @param {PPMSProvider} params.communityCareProvider The community care provider to use. Info aside from
+ *   uniqueId is discarded in version 2
+ * @param {...string} params.fields Other fields provided can be any version 2 field. Some are used to set data
+ *   on the version 0 output (kind, start, etc) and all are merged into the v2 output
+ * @returns {VAOSAppointment|MASAppointment} An appointment object in the specified format
+ */
+export function createMockAppointment({
```

Comment:
I couldn't figure out a good place to put this or how to name it in a way that made it clear why it's different from the other mocks. In any case, we can see how well this mock approach works with some other of these comparison tests and decide if we want to use it more going forward.

Reply:
`createMockAppointmentByVersion`? Is that too on the nose?

I think this is already differentiated by starting the function with `create`, all our other mocks start with `get`.

Reply:
That's a good addition, I'll update the name to that.